setup.py

```diff
@@ -28,6 +28,7 @@
     ],
     entry_points={
         "console_scripts": [
+            "make-naive-message = microcosm_pubsub.main:make_naive_message",
```

Comment:
I'm trying to come up with a more concise CLI name. The best I can come up with is `publish-naive`. Is that better?